cli/tests/test_status_command.rs

```diff
@@ -36,6 +36,33 @@ fn create_commit(
     test_env.jj_cmd_ok(repo_path, &["branch", "create", name]);
 }
 
+#[test]
+fn test_status_copies() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["git", "init", "repo"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("copy-source"), "copy").unwrap();
+    std::fs::write(repo_path.join("rename-source"), "rename").unwrap();
+    test_env.jj_cmd_ok(&repo_path, &["new"]);
+    std::fs::write(repo_path.join("copy-target1"), "copy").unwrap();
+    std::fs::write(repo_path.join("copy-target2"), "copy").unwrap();
+    std::fs::remove_file(repo_path.join("rename-source")).unwrap();
+    std::fs::write(repo_path.join("rename-target1"), "rename").unwrap();
+    std::fs::write(repo_path.join("rename-target2"), "rename").unwrap();
+
+    let stdout = test_env.jj_cmd_success(&repo_path, &["status"]);
+    insta::assert_snapshot!(stdout, @r###"
+    Working copy changes:
+    A copy-target1
+    A copy-target2
+    R {rename-source => rename-target1}
+    A rename-target2
```

Comment:
nit: might be better to simplify the test case to not heavily depend on the git backend behavior.

"copy-target"s aren't detected because the copy-source is unmodified, but ideally they should be.

Reply:
I just meant the test is good so long as it contains at least one rename or copy entry, but the change looks also good to me.

BTW, it might be a bug (in gix or our backend) that `rename-target2` isn't detected as a copy.

Reply:
I simplified the test a bit to only have one rename and one copy.

I agree that `rename-target2` seems like a bug somewhere. I haven't looked into where it is.